subsys/bluetooth/controller/ll_sw/openisa/lll/lll_adv.c

```diff
@@ -608,20 +608,21 @@ static void isr_rx(void *param)
 		crc_ok = radio_crc_is_valid();
 		devmatch_ok = radio_filter_has_match();
 		devmatch_id = radio_filter_match_get();
-		irkmatch_ok = radio_ar_has_match();
-		irkmatch_id = radio_ar_match_get();
+		if (IS_ENABLED(CONFIG_BT_CTLR_PRIVACY)) {
+			irkmatch_ok = radio_ar_has_match();
+			irkmatch_id = radio_ar_match_get();
+		} else {
+			irkmatch_ok = 0U;
+			irkmatch_id = 0xFF;
```

Comment:
Do we have a `#define` for this value? I assume it's some kind of placeholder to denote that there isn't a match?

Reply:
Fixed